index.html

```diff
@@ -13125,7 +13134,7 @@ <h2>Interface Mixin <dfn>AriaAttributes</dfn></h2>
 				attribute FrozenArray&lt;Element&gt;? ariaControlsElements;
 				attribute DOMString? ariaCurrent;
 				attribute FrozenArray&lt;Element&gt;? ariaDescribedByElements;
-				attribute Element? ariaDetailsElement;
+				attribute Element? ariaDetailsElements;
```

Comment:
This would need to be a FrozenArray like ariaDescribedByElements

Reply:
Thanks.